### ISSUE TITLE
Chat progress messages are reported multiple times to screenreader

### ISSUE THREAD

**Issue by roblourens** [microsoft/vscode#211165]
Testing #210661

- Send a request to `@workspace`
- This participant shows 3 progress messages
- The screen reader reads them but they get read multiple times in random order

I'm not scrolling or anything, this is just what happens as the response renders normally. I'm assuming this is due to progress messages being re-rendered, and the screenreader hooking into the rendering flow

**Comment by meganrogge** [comment]
@roblourens I cannot reproduce, I am only seeing the progress messages once and they occur when I expect them to. Can you provide a gif?

**Comment by roblourens** [comment]
They appear once, it's just that the screenreader reads them multiple times

**Comment by meganrogge** [comment]
I don't see the screen reader reading them multiple times. Can you make a gif?

**Comment by roblourens** [comment]
It reads them once as they come in, and a second time when it reads the content of the response when the response is done. Pushing a change to not consider the progress message part of the 'content', which I think is correct since the progress messages disappear from the view once more content has loaded.

This also fixes the fact that the progress messages appear when I Copy a chat response (never noticed that before)
